### ISSUE TITLE
test_text4_unicode2 fails on win32 3.3 and 3.4

### ISSUE THREAD

**Issue by valhallasw** [earwig/mwparserfromhell#78]
Traceback:

```
======================================================================
FAIL: test_text4_unicode2 (tests.test_ctokenizer.TestCTokenizer)
additional unicode check for non-BMP codepoints
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\users\ieuser\documents\github\mwparserfromhell\tests\_test_tokenizer.py", line 56, in inner
    self.assertEqual(expected, actual)
AssertionError: Lists differ: [Text(text='\U00010332\U0001033f\U00010344\U00010330\U00010342\U00010330\U00010336\U00010333\U00010330')] != [Text(text='\ud800\ud800\ud800\ud800\ud800\ud800\ud800\ud800\ud800')]

First differing element 0:
Text(text='\U00010332\U0001033f\U00010344\U00010330\U00010342\U00010330\U00010336\U00010333\U00010330')
Text(text='\ud800\ud800\ud800\ud800\ud800\ud800\ud800\ud800\ud800')

- [Text(text='\U00010332\U0001033f\U00010344\U00010330\U00010342\U00010330\U00010336\U00010333\U00010330')]
+ [Text(text='\ud800\ud800\ud800\ud800\ud800\ud800\ud800\ud800\ud800')]

----------------------------------------------------------------------
Ran 1018 tests in 2.844s

FAILED (failures=1)
```

**Comment by earwig** [comment]
Some weird stuff going on here with surrogate pairs, I think... probably easier for me to just fix #37 and hope that solves it than trying to figure it out.

**Comment by earwig** [comment]
Fixed in 1357da119d37eb893b5de44bfa5ff79ea464cb69 and others.
